### ISSUE TITLE
To many weak opening hands

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#687]
It seems BBA opens on any 11 count
And this is not an opening hand
9CD83EAFA466E279F1391E743C68
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/c6c24956-3eb7-486f-b225-48d4931976f7)

![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/7ac276fb-77e8-41e4-a2f4-2e488d149169)

Can you describe the rules for opening?

**Comment by EdwardPiwowar** [comment]
Any 12 HCP or good 11 HCP or good 10 HCP and good shape. 
In this case, there should be and will be a 2H opening.

**Comment by ThorvaldAagaard** [comment]
I think you should use Any 12, or good 11 with shape and almost never 10 (2-suited and honors in the suits).

**Comment by EdwardPiwowar** [comment]
Perhaps better for Ben, but for BBA it gives a slightly worse result.

**Comment by ThorvaldAagaard** [comment]
Yes, but that is probably because BBA is not playing with real opponents or partners, and using optimistic Single dummy analysis.

BBA is very good for testing this kind of agreements, but you have to have the correct baseline to compare.

So whenever you are letting BBA make non-standard bid in a system, it should be configurable to make sense.

**Comment by EdwardPiwowar** [comment]
BBA uses single dummy analysis primarily for bidding. SD is used to quickly compare results, but double dummy is used for more precise analysis.
As a curiosity:
SD gives a better result if all openings are from 12 (11 + shape).
But you are right that a better comparison would be to other robots, but it takes much more time.

**Comment by ThorvaldAagaard** [comment]
If you can compare 12 + Good 11 with 12 + almost any 11 + good 10, then it should not be that difficult to add it as an option. The problem is that the lighter you open, the more you need to adjust the responses.

I have a common rule, that opening hand + opening hand should give play for game.

**Comment by EdwardPiwowar** [comment]
I can only add new options after switching to the new version of BBA. I'm waiting for you to confirm that it suits you.

**Comment by ThorvaldAagaard** [comment]
yes, I would like to have an option, where opening bids are 12 HCP, or 11 with shape, and an easy rebid, and 10 only with 5+5+ and honors in the suit (again an easy rebid)

When opening with less than 12 there should not be singleton AKQ or J. Doubleton QJ should also be counted as only 2.

So a hand where the longer suit is the lower should normally not open on 11 (but with 4S and a longer minor rebid is often easy)

This is not an opening hand (And I don't like the 6H bid)
08BF00C3128EBE8DF586640FCC71
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/1f4a8451-a871-4490-b8df-d2f8b02c8aea)
1CDC58EDA6A6C314D993600CE447
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/0781b10b-f2be-4f71-b425-5decfbed170a)

This is OK to open
00A67E9213E470902E7782A9CD20
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/08fe2aca-29c8-4424-80ac-037510b75853)
18E87245E3CE1FD111E87BE63B62
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/1cc21b4d-c585-4e5d-bbb2-709a570fe66d)

This is borderline
18DF4643D2CD82C222FB35553020
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/3ca8a89c-4c24-4a34-a4e0-9ff28741120f)
But OK in third hand

This is NOT and opening hand
20A4316997FC5B6AC280DE40CF0F
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/f9f109ef-5b54-4a67-ad21-ac76bdeca213)

11 and a really bad suit
